specification/types.md

```diff
@@ -30,7 +30,16 @@ A structure containing the following fields:
 - reason (string, optional)
 - variant (string, optional)
 
-### evaluation options
+### Flag Resolution
```

Comment:
again, Flag Resolution is ambiguously either a verb or a noun. Would Evaluation Result or Evaluation Outcome be better?

Reply:
How about `ResolutionDetails`? It's consistent with `EvaluationDetails`, and in the context of this type, I don't actually know if the verb/noun confusion is really as problematic. Whether you think of it as a verb or a noun, these are the details associated with it.

I'm going to updated it with `ResolutionDetails`. I'm totally open to changing the term "resolution" entirely. I'd simply like to have a term that denotes the solely the aspect of the flag evaluation that the provider does (in opposition to the entire evaluation which involves hooks, the client, etc).

Reply:
Updated.